src/librustdoc/clean/mod.rs

```diff
@@ -48,11 +48,11 @@ use std::u32;
 
 use parking_lot::ReentrantMutex;
 
-use core::{self, DocContext};
-use doctree;
-use visit_ast;
-use html::render::{cache, ExternalLocation};
-use html::item_type::ItemType;
+use crate::core::{self, DocContext};
```

Comment:
That's ugly. :'(

Reply:
I feel like this shouldn't be necessary with uniform paths?

cc @petrochenkov

Reply:
Although I guess it might be because `core` conflicts with the crate core -- if we rename the module to something that's not a crate then I think we no longer need the `crate::` prefix.